rules/builds.smk

```diff
@@ -35,6 +37,7 @@ rule filter:
             --sequences {input.sequences} \
             --metadata {input.metadata} \
             --include {input.include} \
+            --max-date {input.date} \
```

Comment:
Something went wrong with the rebase, as the definition of `input.date` disappeared for this rule. In any case, I think better to make date a param instead of an input.

Reply:
Ah yes I made an error in rebasing. params.date is defined now. 

I'm curious, what is the distinction between input and params in Snakemake? To me it feels arbitrary because we use entries from `config` in both. I tried looking through the Snakemake docs but couldn't find anything on this. 

After talking with @joverlee521 , my understanding is it that input`= generally is for files and params is for other values.

Reply:
Yeah, I concur with @joverlee521. Snakemake describes the motivation for `params` as well in its docs: https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules